backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Generics_Footer_Report.sql

```diff
@@ -0,0 +1,58 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Generics_Footer_Report(IN p_org_id numeric)
+;
```

Comment:
If you introduce this function, then please used it in ```jasperreports/de/metas/docs/generics/report_footer.jrxml```, in metasfresh repo.

Reply:
done